### ISSUE TITLE
Fix/enable system test: ARIA_invalid_spelling_and_grammar

### ISSUE THREAD

**Issue by feerrenrut** [nvaccess/nvda#13321]
### Steps to reproduce:
As per #13320 system test `ARIA_invalid_spelling_and_grammar` is failing and has been disabled.

Option 1:
Re-enable the test and run with `runsystemtests -i chrome -t ARIA_invalid*`

Option 2:
Manually use NVDA with the following sample:
``` html
<p>Big <span aria-invalid="spelling">caat</span> meos</p>
<p>Small <span aria-invalid="grammar">a dog</span> woofs</p>
<p>Fat <span aria-invalid="grammar, spelling">a ffrog</span> crokes</p>
```
- Press down arrow, note the third line does not report "spelling and grammar error" for `ffrog`

### Actual behavior:
Noted in the test.

### Expected behavior:
Noted in the test.

#### NVDA installed/portable/running from source:
installed / from source

#### NVDA version:
alpha

#### Windows version:
Windows 10

#### Name and version of other software in use when reproducing the issue:
Chrome `100.0.4876.1` and `Version 100.0.4877.2`

#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
N/A
#### If add-ons are disabled, is your problem still occurring?
Yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Not tried.

**Comment by seanbudd** [comment]
I think they have separate goals, one is to ensure our system tests are comprehensive

**Comment by lukaszgo1** [comment]
No, this is not a duplicate. The system test described in this issue used to work but then broke, very likely due to a Chrome upgrade.
